app/templates/live.hbs

```diff
@@ -1,6 +1,19 @@
+{{page-title 'Live'}}
 <LiveHeader
   @tabs={{this.TABS}}
   @activeTab={{this.activeTab}}
   @tabHandler={{this.tabHandler}}
 />
-<p>{{this.activeTab}} Component</p>
\ No newline at end of file
+<div id='live' data-test-live>
+  {{#if (eq this.activeTab 'Screenshare')}}
+    <div class='live__video-container' data-test-video-container>
+      {{#if this.isLoading}}
+        <Reusables::Skeleton />
```

Comment:
If we are adding a Skeleton component to the video component only, we can add it for tabs as well. Just for consistency

Reply:
Right now I have added it for video screen when will layout everything then we will add in others too.
Is it seems good?

Reply:
Sure np. It's just it doesn't looks good only on video screen

Reply:
Yeah I understand